Harvest.Web/ClientApp/src/Requests/AccountsInput.tsx

```diff
@@ -22,6 +23,16 @@ export const AccountsInput = (props: Props) => {
   const typeaheadRef = createRef<AsyncTypeahead<ProjectAccount>>();
 
   const { accounts, setAccounts } = props;
+
+  useEffect(() => {
+    let total = 0;
+    for (let i = 0; i < accounts.length; i++) {
+      total += accounts[i].percentage;
+    }
+
+    total == 100 ? props.setDisabled(false) : props.setDisabled(true);
```

Comment:
just for fun -- I like doing `props.setDisabled(total !== 100)` in these cases instead of using the ternary operator since it is telling you when to set disabled without having to read the ? : logic out